e2e/nomostest/nt.go

```diff
@@ -888,6 +891,47 @@ func (nt *NT) detectGKEAutopilot(skipAutopilot bool) {
 	}
 }
 
+// Prerequisites for autopilot bursting:
+// - You originally created the cluster with GKE version 1.26 or later
+// - The cluster is running GKE version 1.30.2-gke.1394000 or later
+// See: https://cloud.google.com/kubernetes-engine/docs/how-to/pod-bursting-gke#availability-in-gke
+func (nt *NT) autopilotClusterSupportsBursting() (bool, error) {
+	args := []string{"container", "clusters", "describe", nt.ClusterName,
+		"--location", *e2e.GCPRegion, "--format", `value(initialClusterVersion)`}
```

Comment:
The projectID also needs to be provided.

Reply:
Good catch, done